WordPress/src/main/java/org/wordpress/android/ui/jetpackoverlay/JetpackFeatureRemovalOverlayUtil.kt

```diff
@@ -93,13 +95,44 @@ class JetpackFeatureRemovalOverlayUtil @Inject constructor(
         }
     }
 
+    @Suppress("unused", "UNUSED_PARAMETER")
     private fun onFeatureSpecificOverlayShown(feature: JetpackOverlayConnectedFeature) {
-        if (isInFeatureSpecificRemovalPhase())
-            jetpackFeatureOverlayShownTracker.setFeatureOverlayShownTimeStamp(
-                    feature,
-                    getCurrentPhasePreference()!!,
-                    System.currentTimeMillis()
-            )
+        // Commented out for testing purposes only
+//        if (isInFeatureSpecificRemovalPhase())
+//            jetpackFeatureOverlayShownTracker.setFeatureOverlayShownTimeStamp(
+//                    feature,
+//                    getCurrentPhasePreference()!!,
+//                    System.currentTimeMillis()
+//            )
+    }
+
+    @Suppress("MagicNumber")
+    fun initializeFeatureShownOn() {
+        // This function sets the time when the overlay was show
+        // This is only for testing purposes
+
+        // The first parameter sets the no of days when the overlay was last shown
+        // For example, value 2 means, the overlay shown time stamp will be set 2 days before
+        // The Second parameter sets the feature for which the overlay was shown
+
+        setFeatureAccessedOn(3, JetpackOverlayConnectedFeature.STATS)
+        setFeatureAccessedOn(2, JetpackOverlayConnectedFeature.NOTIFICATIONS)
+        setFeatureAccessedOn(4, JetpackOverlayConnectedFeature.READER)
+
```

Comment:
@JB184351  you can comment out or change the values on these lines to test the frequency logic. 

Let me know if you need any additional info

Reply:
Awesome, thank you! Will do that today!

Reply:
I'm trying to change the values but I don't see it applying, I'm not used to running Android Studio so maybe I'm doing something wrong there, but should running the app again apply the code changes? I'm seeing different options that might enable these changes. @AjeshRPai

Reply:
Hey @JB184351 

>but should running the app again apply the code changes?
So I feel I missed giving you more info. Sorry for that. 

When u want to change the values. You might have to run the app by 
1. commenting the lines from `118-120`  
2. uncomment the line `123`        `// jetpackFeatureOverlayShownTracker.clear()` 
3. run the app. 

This will clear all the values. Then to change the values(when the feature was last accessed) , do the reverse. 

1. Uncomment lines from 118-120
6. Comment line 123

You can comment or uncomment the code by putting "//" before each line. 

Let me know if you want more information or any help with it

Reply:
Ok, I tried this uncommenting line 123 for when trying to change values but I get this error trying to run the app.
<img width="1916" alt="Screen Shot 2022-11-17 at 1 30 16 PM" src="https://user-images.githubusercontent.com/36495451/202553282-5e72ee2c-df3c-4335-b185-d710f1f20edc.png">

Reply:
@JB184351 So sorry for that. Sometimes I make stupid mistakes. 😅 . I removed the `jetpackFeatureOverlayShownTracker.clear()` by mistake on this branch when I removed it from the Original PR. I have added it again with this [commit](https://github.com/wordpress-mobile/WordPress-Android/pull/17475/commits/5db0e7f874ee1dccc1aeb6754fe9497e8b342fdc). 

Please take a pull of the changes

Reply:
Ok I pulled it, so if I wanted to change the time to 2 minutes would I just changed the constant on line 22 to `120000L`? Like this? `private const val ONE_DAY_TIME_IN_MILLIS = 120000L`? Though it looks like it's taking the current system time into account when making the calculation as well. @AjeshRPai. Sorry for all the questions 😅

Reply:
>Sorry for all the questions 

No worries Justin.

 So In Android, we have kept the frequency of the overlay logic in days. So it's not possible to check the overlay logic using Minutes/Seconds logic as in IOS.  What we can do here is to change the date on which the overlay was last shown. 

The first parameter in `setFeatureAccessedOn` is the no of days elapsed since the feature overlay was shown). So 3 means, it's been 3 days since the feature was accessed. This way when the app launches since the feature-specific overlay for the first phase is 2 this way the feature-specific overlay should be shown as per our requirements. 

`setFeatureAccessedOn(3(days since overlay was shown), JetpackOverlayConnectedFeature.STATS(feature which was accessed))
setFeatureAccessedOn(2, JetpackOverlayConnectedFeature.NOTIFICATIONS)
setFeatureAccessedOn(4, JetpackOverlayConnectedFeature.READER)`

Let me know if this makes sense/ if you need any additional info.

Reply:
mmm ok, so how would I change it to be only 1 day? Would I just change this variable `val featureAccessedMockedTimeinMillis = (System.currentTimeMillis() -
                (noOfDaysPastFeatureAccessed * ONE_DAY_TIME_IN_MILLIS))` to   `val featureAccessedMockedTimeinMillis = (System.currentTimeMillis() -
                (ONE_DAY_TIME_IN_MILLIS))`?